dbt/macros/test_analytics/store_test_results.sql

```diff
@@ -0,0 +1,99 @@
+/*
+  --add "{{ store_test_results(results) }}" to an on-run-end: block in dbt_project.yml 
+  --The next v.1.0.X release of dbt will include post run hooks for dbt test! 
+*/
```

Comment:
```suggestion
#}
```
These should probably be jinja comments?